Angular-Make-Useful-APIs.md

```diff
@@ -1,19 +1,22 @@
-There are two more things you have to do before this to be useful to you, however. Say you want to show all the _things_ associated with the username requested with that page: you must first  
-1. Have a "username" or "owner" field in your _thing_ schema at **/server/api/thing/thing.model.js**  
-2. Write a custom route in **/server/api/thing/index.js** to catch a request for a specific username. The request from your frontend might look something like:
+# Make Useful APIs
```

Comment:
Just add "Angular" here, like Angular : Make Useful APIs or Make Useful APIs in Angular. adds a bit context